audio/mpd/Portfile

```diff
@@ -28,9 +28,9 @@ master_sites        https://www.musicpd.org/download/${name}/${branch}/
 license             GPL-2+ LGPL-2.1
 use_xz              yes
 
-checksums           rmd160  c288933cd369d606e8fdfe49515c67cadc3da56e \
-                    sha256  503e5f9f237290f568ff7956ab2f9aed563594bf749f19b8fe994fb21434afea \
-                    size    806784
+checksums           rmd160  9951bf096e8d6d6fe3a07a82d660caf1d2bd91af \
+                    sha256  20a0ad01bf327b2dbeb6ae8e1af952cb0de83d2d63fab0fa4b7183a74765c201 \
+                    size    683768
 
 depends_build \
     port:pkgconfig
```

Comment:
Please also change this to:
```
depends_build-append \
    port:pkgconfig
```

`depends_build` (without `-append`) removes meson and other build dependencies specified by the `meson 1.0` portgroup, causing the CI builds to fail:
```
sh: /opt/local/bin/meson: No such file or directory
```

Reply:
Cool, thanks! Turns out there's a lot more changes for the meson build.